packages/core/src/components/tag/tag.tsx

```diff
@@ -35,7 +35,8 @@ export class Tag extends React.Component<ITagProps, {}> {
         return (
             <span {...removeNonHTMLProps(this.props)} className={tagClasses}>
                 {this.props.children}
-                {isFunction(onRemove) ? <button className={Classes.TAG_REMOVE} onClick={onRemove} /> : null}
+                {isFunction(onRemove) ? <button type="button" className={Classes.TAG_REMOVE} onClick={onRemove} />
+                : null}
```

Comment:
I was doing just that. Also, I'm sorry, I added PR metadata to the commit message before I should have.